src/Dialect/ONNX/ONNXOps/Canonicalize.cpp

```diff
@@ -584,6 +585,66 @@ class PropagateReshapeThroughBinaryOpPattern
   };
 };
 
+// This pattern bubbles up AddOp through transpose to keep the bias Add
+// operation right after LN_type op. This will helps the other patterns fold the
+// add into the operands of LayerNorm.
```

Comment:
NIT: not only Layernorm
```suggestion
// add into the operands of a Norm operator
```